arangod/RestHandler/RestStatusHandler.cpp

```diff
@@ -50,6 +51,17 @@ RestStatusHandler::RestStatusHandler(GeneralRequest* request, GeneralResponse* r
     : RestBaseHandler(request, response) {}
 
 RestStatus RestStatusHandler::execute() {
+  ServerSecurityFeature* security =
+      application_features::ApplicationServer::getFeature<ServerSecurityFeature>(
+          "ServerSecurity");
+  TRI_ASSERT(security != nullptr);
+
+  if (!security->canAccessHardenedApi()) {
```

Comment:
this is not necessary for the Foxx API toggles.

Reply:
Additionally, the method `canAccessHardenedApi` is not a member of the `ServerSecurityFeature` class above.

Reply:
should not have made it to github